### PR TITLE
Added documentation page for Multiplayer Use Cases Sample

### PR DESCRIPTION
**Pull request by RikuTheFuffs**
- - -
**Select the type of change:**
- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Added documentation page for Multiplayer Use Cases Sample

**Issue Number:**
- https://jira.unity3d.com/browse/MTT-6007 (it's the "companion docs" part)

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=1326) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=1326) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=1326) it.</sub>

**Comment by Vic-Cooper** [comment]
Made changes to make the page function on the site, and made some basic improvements to the content. 
I'm on PTO until Tuesday so I'd like to have another look at it then if there's no time pressure to publish this PR before then. 

If there is, publish it in this state and I'll review it again later :)

File: docs/learn/bitesize/bitesize-usecases.md [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
-The In-Editor tutorials are powered by the official [Tutorial Framework package](https://docs.unity3d.com/Packages/com.unity.learn.iet-framework@4.0/manual/index.html) and can be accessed at any times from the "Tutorials" menu.
+## The NetvarVsRpc scene
 
-It is highly recommended to follow the In-Editor tutorials to get the most value when exploring this Sample.
+The NetvarVsRpc scene demonstrates the differences betweeen the way NetworkVariables and Remote Procedure Calls (RPCs) perform state synchronization.
 
+## The NetworkVariables scene
 
-## Scene: "NetvarVsRpc"
+The NetworkVariables scene demonstrates the way NetworkVariables perform state synchronization in a way that sends the most recent information to late joining or reconnecting clients.
```
Comment (RikuTheFuffs) [inline docs/learn/bitesize/bitesize-usecases.md:26]: @Vic-Cooper  on this one, I'd add an "also" between "that" and "sends", to clarify that this is an additional behaviour (not the only one)

Comment (Vic-Cooper) [reply docs/learn/bitesize/bitesize-usecases.md]: Done in latest commit!

**Review by Vic-Cooper — APPROVED** [review]

**Pull request merged**